pkg/eestream/crc_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package eestream
+
+import (
+	"encoding/binary"
+	"hash/crc32"
+
+	"storj.io/storj/pkg/eestream/ranger"
+)
+
+type crcAdder struct {
+	Table *crc32.Table
+}
+
+func newCRCAdder(t *crc32.Table) *crcAdder {
+	return &crcAdder{Table: t}
+}
+
+func (c *crcAdder) InBlockSize() int  { return 64 }
+func (c *crcAdder) OutBlockSize() int { return 64 + 4 + 8 }
```

Comment:
I would appreciate some description of these magic numbers.

Reply:
good point will do